File: src/WebApiHelpPage/Areas/HelpPage/ModelDescriptions/EnumTypeModelDescription.cs
using System.Collections.Generic;

namespace ROOT_PROJECT_NAMESPACE.Areas.HelpPage.ModelDescriptions
{
    public class EnumTypeModelDescription : ModelDescription
    {
        public EnumTypeModelDescription()
        {
            Values = new List<EnumValueDescription>();
        }

        public IList<EnumValueDescription> Values { get; private set; }
    }
}

File: src/WebApiHelpPage/VB/Areas/HelpPage/ModelDescriptions/EnumTypeModelDescription.vb
Imports System.Collections.Generic

Namespace Areas.HelpPage.ModelDescriptions
    Public Class EnumTypeModelDescription
        Inherits ModelDescription
        Private _values As IList(Of EnumValueDescription)

        Public Sub New()
            Values = New List(Of EnumValueDescription)()
        End Sub

        Public Property Values() As IList(Of EnumValueDescription)
            Get
                Return _values
            End Get
            Private Set(value As IList(Of EnumValueDescription))
                _values = value
            End Set
        End Property
    End Class
End Namespace